src/devices/vdml_gps.c

```diff
@@ -101,7 +146,7 @@ double gps_get_heading_raw(uint8_t port) {
 
 double gps_get_rotation(uint8_t port) {
 	claim_port_f(port - 1, E_DEVICE_GPS);
-	double rtv = vexDeviceGpsRotationGet(device->device_info);
+	double rtv = vexDeviceGpsHeadingGet(device->device_info);
```

Comment:
I don't exactly approve of this change, it gives users less options and also doesn't fit the VEX API loose rule of having total rotation as simply "rotation" whereas heading and angle are used as the 360 constrained rotation.

Reply:
That was the intention of the get_rotation the call we were using before returning the initial set rotation. If a user set rotation to 90 degrees then gps_get_rotation would only return that value and if the user didn't set it it would return 0. The previous behavior was more get_intial_rotation. The new method returns the live unbounded rotation unconstrained

Reply:
Interesting, we may want to contact VEX about this actually. This seems to be more of a bug than an intended feature.